releasenotes/notes/add-prettyprint-8ec601f41c9cc7d2.yaml

```diff
@@ -0,0 +1,95 @@
+---
+upgrade:
+  - |
+    Shows a warning message if non-printable strings are set to
+    :class:`~qiskit_optimization.problems.QuadraticProgram`
+    as problem name, variable name, or constraint name.
+  - |
+    If users set an empty variable name ``""`` with
+    :meth:`~qiskit_optimization.problems.QuadraticProgram.binary_var`,
+    :meth:`~qiskit_optimization.problems.QuadraticProgram.integer_var`, and
```

Comment:
I wish I could see why these do not seem to be coming out as links in the html. They are highlighted but no clickable as if it failed to resolve them but I cannot see why.

Reply:
Yeah, I cannot see the links to `QuadraticProgram` and methods `binary_var`, etc., either.
Do you have any idea, @manoelmarques?

Reply:
I figured why - we have import the class to the root and document it there ie at `~qiskit_optimization.QuadraticProgram`

Reply:
The references now work - I made one small final change to the text.

Reply:
Thanks!